mlir/unittests/TableGen/PassGenTest.cpp

```diff
@@ -72,7 +72,7 @@ TEST(PassGenTest, PassOptions) {
   TestPassWithOptionsOptions options;
   options.testOption = 57;
 
-  llvm::SmallVector<int64_t, 2> testListOption = {1, 2};
+  llvm::SmallVector<int64_t> testListOption = {1, 2};
   options.testListOption = testListOption;
```

Comment:
```suggestion
  options.testListOption = {1, 2};
```
Do you even need the temporary anymore?